### ISSUE TITLE
Write train.py

### ISSUE THREAD

**Issue by archemist-hong** [archemist-hong/pytorch-template#3]
## Background
- Dataset, DataLoader, Optimizer, Loss, Model 등 config 파일로부터 생성
- MNIST데이터셋을 활용한 Multi-layer perceptron으로 train.py 완성
- 훈련과정 wandb 기록, slack 메신저 전송 필요

## ToDo
- [ ] yaml파일로 Model을 비롯한 객체 생성 코드 작성
- [ ] 모델 train/validation 코드 작성
- [ ] 훈련과정 wandb 기록 logging 기능 추가
- [ ] slack 메신저로 전송 기능 추가
